src/nodes/mix-node-setup.md

```diff
@@ -501,6 +501,43 @@ The code for exposed API endpoints can be found [here](https://github.com/nymtec
 
 > You can get more detailed info by appending `?debug` to the URL, like so: `curl http://<NODE_IP_ADDRESS>:8000/stats?debug`
 
+### Mix node Reward Estimation API endpoint
```

Comment:
this should be in the section below ( `Mix node related Validator API endpoints ` ) - put it in a subheading there and mention that this is a particularly common one that operators would want to know about